libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -351,3 +357,21 @@ finishedPulserState ::
   NewEpochState era ->
   (PulsingSnapshot era, RatifyState era)
 finishedPulserState nes = finishDRepPulser (nes ^. newEpochStateGovStateL . drepPulsingStateGovStateL)
+
+-- | Query a stake pool's reward account delegatee which determines the pool's default vote
+-- in absence of an explicit vote. Note that this is different from the delegatee determined
+-- by the credential of the stake pool itself.
+querySPODefaultVote ::
+  NewEpochState era ->
+  -- | Specify the key hash of the pool whose default vote should be returned.
+  KeyHash 'StakePool (EraCrypto era) ->
+  -- | Depending on the result, the default votes are as follows:
+  --     * @Just DRepAlwaysAbstain@: default vote is @Abstain@
+  --     * @Just DRepAlwaysNoConfidence@: default vote is @Yes@ in case of a @NoConfidence@ action,
+  --       otherwise @No@
+  --     * Other (@Nothin@ or @DRepKeyHash@, @DRepScriptHash@): default vote is @No@
+  -- In case of a @HardForkInitiation@, the default vote is always @No@.
+  Maybe (DRep (EraCrypto era))
```

Comment:
We need to create a default vote type, instead of returning a DRep, because obtaining a DRep that StakePool's reward account delegated to, does not mean much, unless it is a predefined DRep. So, we should make it for the user absolutely clear through a specialized type what is going on.
```suggestion
  DefaultVote
```
where
```haskell
data DefaultVote =
  = DefaultNo
  | DefaultAbstain
  | DefaultNoConfidence
```

Reply:
Sounds good! :+1: I was thinking about this as well but since I was not sure if we want to do it, I opted to provide an explanation in the docs as a first draft instead. I'll make the change.

Reply:
I added the data type in this module and exported it in case someone downstream wants to use it. Let me know if you want the type to be defined somewhere else or if you think we shouldn't expose the data type to users.